tests/Microsoft.Identity.Web.Test/WebApiExtensionsTests.cs

```diff
@@ -99,9 +99,7 @@ private void AddProtectedWebApi_TestCommon(IServiceCollection services, ServiceP
             Assert.Equal(ServiceLifetime.Singleton, services.First(s => s.ServiceType == typeof(IJwtBearerMiddlewareDiagnostics)).Lifetime);
 
             // JWT options added correctly
-            var configuredJwtOptions = provider.GetService<IConfigureOptions<JwtBearerOptions>>() as ConfigureNamedOptions<JwtBearerOptions>;
-
-            Assert.Equal(_jwtBearerScheme, configuredJwtOptions.Name);
+            var configuredJwtOptions = provider.GetService<IConfigureOptions<JwtBearerOptions>>() as IConfigureNamedOptions<JwtBearerOptions>;
```

Comment:
- This is an issue w/.NET 5, it's a `ConfigureNamedOptions<JwtBearerOptions, Microsoft.AspNetCore.Authentication.JwtBearer>`, which is not a `ConfigureNamedOptions<JwtBearerOptions>. and, Microsoft.AspNetCore.Authentication.JwtBearer` is not a public type.
 
`x.GetType().FullName
"Microsoft.Extensions.Options.ConfigureNamedOptions`2[[Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerOptions, Microsoft.AspNetCore.Authentication.JwtBearer, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60],[System.IServiceProvider, System.ComponentModel, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a]]"`

We probably don't need to check the `name`, this isn't surfaced by the interface.